src/simtools/applications/plot_tabular_data.py

```diff
@@ -56,7 +57,7 @@ def _parse(label, description, usage):
         type=str,
         required=True,
     )
-    return config.initialize(db_config=True)
+    return config.initialize(db_config=True, simulation_model=["telescope"])
```

Comment:
If I dig deep I might find an answer to this question, but do we really need the telescope models for every call of this application? The application name is quite general and just from it I would be a bit surprised of this behaviour.

(Technically that is also true for the DB connection, but never mind.)

Reply:
No, this is an open issue that the naming of our configuration sections should be updated: this adds the "--telescope" as a command line option. This should be changed to an 'Instrument' section (or something similar). On the todo list...also related to the other points in #1055. No telescope model is needed!

Reply:
OK, the question still stands as to why we need the `--telescope` command line option for every call to this application, but it's not a blocking point.

Reply:
It is optional, so you don't have to give it (look e.g., at tests/integration_tests/config/derive_photon_electron_spectrum_lst_ecsv.yml).

This is used only to get a minimum set of metadata.